internal/datasource/webclient/cloudproviders/genqlient.graphql

```diff
@@ -10,8 +10,8 @@ mutation ProvisionCertificate(
     }
 }
 
-query GetCloudKeystoresByKeystoreId($cloudKeystoreId: UUID) {
-    cloudKeystores(filter: {cloudKeystoreId: $cloudKeystoreId}) {
+query GetCloudKeystoresByKeystoreInfo($cloudKeystoreId: UUID, $cloudKeystoreName: String, $cloudProviderId: UUID, $cloudProviderName: String) {
```

Comment:
can this be called just `GetCloudKeystores`? You are providing all the filters so no need to be specific

Reply:
No problem

Reply:
Done by latest changes